content/blog/2025-05-08-go-client-GA.md

```diff
@@ -0,0 +1,519 @@
++++
+title= "Valkey GLIDE Go Client: Now Generally Available!"
+description = "Valkey GLIDE Go client reaches general availability. Read to learn more about the go client designed for performance and developer productivity"
+date= 2025-05-08 01:01:01
+authors= [ "niharikabhavaraju", "jbrinkman"]
++++
+
+The Valkey GLIDE contributors are pleased to announce the general availability (GA) release of the Valkey GLIDE Go client. This release brings the power and reliability of Valkey to Go developers with an API designed for performance and developer productivity.
+
+Vakey GLIDE 2.0 brings exciting new features for all of our language clients (Java, Node.js, Python, and now Go). These features include pipelines, OpenTelemetry support, MacOS x86 support, improved performance, and bug fixes.
```

Comment:
I would cut this. It seems like maybe this is worth a separate blog post, but it's not really relevant to this blog.